int.ts

```diff
@@ -911,48 +911,48 @@ module J2ME {
             sp -= 2;
             continue;
           case Bytecodes.DUP: // ... a -> ... a, a
-            i32[sp] = i32[sp - 1];          ref[sp] = ref[sp - 1];
+            i32[sp] = i32[sp - 1];          i32[sp] = i32[sp - 1];
```

Comment:
I admit I've simply blindly replaced "ref" with "i32", without actually looking at the code. I'll clean the patch up when I fix tests.